app/helpers/react_helper.rb

```diff
@@ -1,13 +1,15 @@
 # frozen_string_literal: true
 
 module ReactHelper
+  # rubocop:disable Metrics/AbcSize
```

Comment:
I guess it's fine right here since all this will be API driven in the future 🤞🏽 , no need to do a refactor of this right now.